### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by btjanaka**
## TODO

- [x] Update optimization image in README
- [x] Update code in README
- [x] Update installation command in README

## Status

- [x] I have read the guidelines in [CONTRIBUTING.md](https://github.com/icaros-usc/pyribs/blob/master/CONTRIBUTING.md)
- [x] I have formatted my code using `yapf`
- [x] I have tested my code by running `pytest`
- [x] I have linted my code with `pylint`
- [x] I have added a one-line description of my change to the changelog in `HISTORY.md`
- [x] This PR is ready to go

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,8 +5,10 @@
...
 
 A _bare-bones_ Python library for quality diversity optimization. pyribs is the
-official implementation of the Covariance Matrix Adaptation MAP-Elites (CMA-ME)
-algorithm and implements the _Rapid Illumination of Behavior Space (RIBS)_
+official implementation of Covariance Matrix Adaptation MAP-Elites (CMA-ME),
+Covariance Matrix Adaptation MAP-Elites via a Gradient Arborescence (CMA-MEGA),
+Covariance Matrix Adaptation MAP-Annealing (CMA-MAE), and scalable variants of
+CMA-MAE. pyribs implements the _Rapid Illumination of Behavior Space (RIBS)_
 redesign of MAP-Elites detailed in the paper
 [Covariance Matrix Adapation for the Rapid Illumination of Behavior Space](https://arxiv.org/abs/1912.02400).
```
Comment (itsdawei) [inline README.md:13]: I think we either link all the papers here or don't link any of the papers -- to keep it consistent

File: README.md [hunk 2]
```diff
@@ -5,8 +5,10 @@
...
 
 A _bare-bones_ Python library for quality diversity optimization. pyribs is the
-official implementation of the Covariance Matrix Adaptation MAP-Elites (CMA-ME)
-algorithm and implements the _Rapid Illumination of Behavior Space (RIBS)_
+official implementation of Covariance Matrix Adaptation MAP-Elites (CMA-ME),
+Covariance Matrix Adaptation MAP-Elites via a Gradient Arborescence (CMA-MEGA),
+Covariance Matrix Adaptation MAP-Annealing (CMA-MAE), and scalable variants of
+CMA-MAE. pyribs implements the _Rapid Illumination of Behavior Space (RIBS)_
```
Comment (itsdawei) [inline README.md:11]: "pyribs also implements..."

**Review by itsdawei — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (btjanaka) [reply README.md:13]: We link to CMA-ME since it has the original design, then we link all the papers below

File: README.md (hunk 2 above)
Comment (btjanaka) [reply README.md:11]: I'll clarify this.

**Pull request merged**